src/app/issuer/models/badgeclass.model.ts

```diff
@@ -42,6 +42,13 @@ export class BadgeClass extends ManagedEntity<ApiBadgeClass, BadgeClassRef> {
 		this.apiModel.image = image;
 	}
 
+	get imageFrame(): boolean {
+		return this.apiModel.imageFrame;
```

Comment:
Do you know what happens if `imageFrame` is not defined (since it's an optional property)? I assume it would yield something falsey; maybe it would make more sense to then also implement a default value (as in the backend) of `true`. So:
```ts
get imageFrame(): boolean {
  return this.apiModel.imageFrame !== undefined ? this.apiModel.imageFrame : true;
}
```
or sth similar